### PR TITLE
Update guide-chapter-1.md

### PR DESCRIPTION
**Pull request by vtchem**
## Which rule and warnings did you add?

This pull request closes 

## Actions checklist
* [ ] Implemented Rule, added Warnings
* [ ] Added tests on checks
* [ ] Added tests on fixers
* [ ] Updated diktat-analysis.yml
* [ ] Updated available-rules.md

### CODE REVIEW

File: info/guide/guide-chapter-1.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # <a name="c1"></a> 1. Naming
-In programming, it is not always easy to meaningfully and appropriately name variables, functions, classes, etc. Using meaningful names helps to clearly express your code's main ideas and functionality and avoid misinterpretation, unnecessary coding and decoding, "magic" numbers, and inappropriate abbreviations.
+It is not always easy to meaningfully and appropriately name variables, functions, classes, and so on. Using meaningful names in programming helps to clearly express the main ideas and functionality of your code and avoid its misinterpretation, unnecessary coding and decoding, "magic" numbers, and inappropriate abbreviations.
 
 Note: The source file encoding format (including comments) must be UTF-8 only. The ASCII horizontal space character (0x20, that is, space) is the only permitted whitespace character. Tabs should not be used for indentation.
```
Comment (vtchem) [inline info/guide/guide-chapter-1.md:4]: This is not clear, please explain:
The ASCII horizontal space character (0x20, that is, space)..

Comment (0x6675636b796f75676974687562) [reply info/guide/guide-chapter-1.md:4]: https://en.wikipedia.org/wiki/ASCII#Printable_characters

Comment (vtchem) [reply info/guide/guide-chapter-1.md:4]: My question was: What is meant by: “(0x20, that is, space)”? Could you, please, rephrase it?.

Comment (0x6675636b796f75676974687562) [reply info/guide/guide-chapter-1.md:4]: I'd put it as 

> Source code files (incl. comments) must be UTF-8 encoded, no exceptions. The ASCII _horizontal space_ (`0x20`, `U+0020`) is the only permitted whitespace character. _Tab_ character (`0x09`, `U+0009`) should never be used for indentation.

Does it make more sense now?

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

File: info/guide/guide-chapter-1.md (hunk 1 above)
Comment (vtchem) [reply info/guide/guide-chapter-1.md:4]: OK. I changed it.

**Pull request merged**